db/migrate/20180517205153_remove_conference_preferences_without_team.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveConferencePreferencesWithoutTeam < ActiveRecord::Migration[5.1]
```

Comment:
Should there be a validation for `team_id` presence on `ConferencePreference`?

Reply:
Yes, seem my comment earlier. All `belongs_to` associations should be required by default....

Reply:
Missed it! Thank you for pointing it out - makes sense 👍

Reply:
😹 yes, below. I'm usually processing my notifications in chronological order 🕐